sp800-63a/sec4_ial.md

```diff
@@ -6,7 +6,7 @@
 
 _This section is normative._
 
-This document describes the common pattern in which an applicant undergoes an identity proofing and enrollment process in which their identity evidence and attributes are collected, uniquely resolved to a single identity within a given population or context, then validated and verified. A CSP may then bind these attributes to an authenticator (described in [[SP 800-63B]](sp800-63b.html)).
+This document describes the common pattern in which an applicant undergoes an identity proofing and enrollment process in which their identity evidence and attributes are collected, uniquely resolved to a single identity within a given population or context, then validated and verified. See [Selecting IAL](https://pages.nist.gov/TIG-Stage/sp800-63-3.html#IAL_CYOA) for details on how to chose the most appropriate IAL. A CSP may then bind these attributes to an authenticator (described in [[SP 800-63B]](sp800-63b.html)).
```

Comment:
Need to show document and section number, not just [Selecting IAL], so that it can be seen in printed documents. Also need to use a relative URL, not point specifically to TIG-Stage.